### PR TITLE
Add additional comments and examples for deploy and undeploy commands

### PR DESCRIPTION
**Pull request by NateThornton**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The update.sh command will switch that submodule back to the head of its master
 
 ## nnf-deploy
 
-nnf-deploy is a golang executable capable of building all of the docker components of the Rabbit software stack locally as well as deploying and undeploying those components to a k8s cluster specified by the current kube config.
+nnf-deploy is a golang executable capable of building components of the Rabbit software stack locally as well as deploying and un-deploying those components to a k8s cluster specified by the current kube config.
```
Comment (ajfloeder) [inline Readme.md:37]:
```suggestion
nnf-deploy is a golang executable capable of building components of the Rabbit software stack locally as well as deploying and un-deploying those components to a k8s cluster specified by the current kubeconfig.
```

File: Readme.md [hunk 2]
```diff
@@ -96,20 +96,38 @@ Once the labels/taint are applied, cert manager will be installed.
...
 ## Deploy
 
-Deploying will deploy all the submodules to your current kube config context
+Deploy all the submodules using the `deploy` command
 
 ```bash
 ./nnf-deploy deploy
 ```
 
+To only deploy specific repositories, include the desired modules after `deploy` command. For example, to only deploy `dws` and `nnf-sos` repositories, use
```
Comment (ajfloeder) [inline Readme.md:105]:
```suggestion
To deploy only specific repositories, include the desired modules after `deploy` command. For example, to deploy only `dws` and `nnf-sos` repositories, use
```

File: Readme.md [hunk 3]
```diff
@@ -96,20 +96,38 @@ Once the labels/taint are applied, cert manager will be installed.
...
+```bash
+./nnf-deploy deploy dws nnf-sos
+```
+
 ## Undeploy
 
-Undeploy all the submodules
+**WARNING!** Undeploy should be preceded by deleting any user or administrator created resources such as `lustrefilesystems` and `workflows` using kubectl commands
```
Comment (ajfloeder) [inline Readme.md:112]:
```suggestion
**WARNING!** Before you undeploy, delete any user or administrator created resources such as `lustrefilesystems` and `workflows` using kubectl commands
```

File: Readme.md [hunk 4]
```diff
@@ -96,20 +96,38 @@ Once the labels/taint are applied, cert manager will be installed.
...
+kubectl delete lustrefilesystems.cray.hpe.com --all
+```
+
+Undeploy all the submodules using the `undeploy` command.
 
 ```bash
 ./nnf-deploy undeploy
 ```
 
+Similar to deploy, undeploy specific repositories by including the desired modules after the `undeploy` command. For example, to only undeploy `dws` and `nnf-sos`, use
```
Comment (ajfloeder) [inline Readme.md:125]:
```suggestion
Similar to deploy, you may undeploy specific repositories by including the desired modules after the `undeploy` command. For example, to undeploy only `dws` and `nnf-sos`, use
```

**Review by ajfloeder — APPROVED** [review]

**Review by bdevcich-hpe — APPROVED** [review]

File: Readme.md [hunk 5]
```diff
@@ -96,20 +96,38 @@ Once the labels/taint are applied, cert manager will be installed.
...
 
 ```bash
 ./nnf-deploy deploy
 ```
 
+To deploy only specific repositories, include the desired modules after `deploy` command. For example, to deploy only `dws` and `nnf-sos` repositories, use
+```bash
+./nnf-deploy deploy dws nnf-sos
+```
+
```
[the remark this answers is not in the document]
Comment (NateThornton) [reply Readme.md:109]: arguments can be in any order. The tool itself deploys in the order of the `modules` array

**Pull request merged**